ckanext/dashboard/blueprints/dashboard.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from ckan import model
+import ckan.plugins as p
+from flask import Blueprint, request, redirect, url_for, flash
+from ckan.logic import NotFound
+from ckan.plugins.toolkit import render
+
+# Importa o define el decorador para restringir el acceso a sysadmins.
+# Puedes definirlo en tu extensión o importarlo si ya lo tienes.
+from ckanext.dashboard.decorators import require_sysadmin_user
+
+log = logging.getLogger(__name__)
+
+dashboard_bp = Blueprint('dashboard_bp', __name__, url_prefix='/dashboard_2')
```

Comment:
```suggestion
dashboard_bp = Blueprint('dashboard_bp', __name__, url_prefix='/dashboard-external')
```

`-` is bettar than `_` for URLs
`2` is not clear